### PR TITLE
Add color to output

### PR DESCRIPTION
**Pull request by ulziibuyan**
#5

### CODE REVIEW

File: src/sinonimos/main.go [hunk 1]
```diff
@@ -2,6 +2,8 @@ package main
 
 import (
 	"fmt"
+	. "github.com/logrusorgru/aurora"
```
[the remark this answers is not in the document]
Comment (ulziibuyan) [reply src/sinonimos/main.go]: @felipemfp Should I modify the code to not use "dot imports"?

[the remark this answers is not in the document]
Comment (felipemfp) [reply src/sinonimos/main.go]: Don't worry. It's fine!

**Comment by felipemfp** [comment]
@ulziibuyan, could you rebase your changes?

```bash
git pull origin master --rebase
```

**Comment by felipemfp** [comment]
Thank you, @ulziibuyan. Unfortunately we have to manually merge it:  https://github.com/felipemfp/sinonimos-cli/commit/dfbc0a330b64ef9486da445b2b4a79cf1527ccd6

:tada:

**Pull request closed without merging**